docs/sections/pipeline_samples/tutorials/generate_textcat_dataset.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        ),
</pre><p>perhaps we can remove this step? I think it is not required to understand the flow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>